package.json

```diff
@@ -8,11 +8,12 @@
     "serve": "vue-cli-service serve",
```

Comment:
PS The package-lock file has a lot of changes because of the package updates (and because I deleted and re-created it, so there's no left-over package dependencies in it, yay!).